README.rst

```diff
@@ -40,18 +68,52 @@ These are the dependencies to use ``autoreject``:
 
 Optional dependencies are:
 
-* ``tqdm`` (for nice progressbars)
-* ``h5py`` (for IO)
-* ``openneuro-py`` (for fetching data from OpenNeuro.org)
+* ``tqdm`` (for nice progressbars when setting ``verbose=tqdm``)
+* ``h5py`` (for writing ``autoreject`` objects using the HDF5 format)
+* ``openneuro-py`` (>= 2021.7, for fetching data from OpenNeuro.org)
 
-Cite
-----
+Quickstart
+==========
 
-If you use this code in your project, please cite::
+The easiest way to get started is to copy the following three lines of code
+in your script:
+
+.. code:: python
+
+	>>> from autoreject import AutoReject
+	>>> ar = AutoReject()
+	>>> epochs_clean = ar.fit_transform(epochs)  # doctest: +SKIP
+
+This will automatically clean an `epochs` object read in using MNE-Python. To get the
+rejection dictionary, simply do:
+
+.. code:: python
+
+	>>> from autoreject import get_rejection_threshold
+	>>> reject = get_rejection_threshold(epochs)  # doctest: +SKIP
+
+We also implement RANSAC from the PREP pipeline.
+The API is the same:
+
+.. code:: python
+
+	>>> from autoreject import Ransac
+	>>> rsc = Ransac()
+	>>> epochs_clean = rsc.fit_transform(epochs)  # doctest: +SKIP
+
+For more details check out the :ref:`example to automatically detect and repair bad epochs <sphx_glr_auto_examples_plot_auto_repair.py>`.
```

Comment:
This seems broken

Reply:
Yes, this will not work in the GitHub rendering, only in the actual docs.

Should I remove the link? Do you have another idea?

Reply:
You can link with the URL for the dev version. I think that should be good enough